lib/Cargo.toml

```diff
@@ -56,6 +56,9 @@ static_assertions = { workspace = true }
 default = ["install"]
 # This feature enables `bootc install`.  Disable if you always want to use an external installer.
 install = []
+# This featuares enables `bootc internals publish-rhsm-facts` to integrate with
+# Red Hat Subscription Manager
+rhsm = []
```

Comment:
OK as is, but we will also need to do a corresponding change to the Fedora spec file to turn it on (bcond that is default off in Fedora, on in RHEL?)